src/Fusion.Summary.Api/Controllers/DepartmentsController.cs

```diff
@@ -43,31 +36,19 @@ public async Task<IActionResult> GetDepartmentsV1()
 
         #endregion Authorization
 
-        var ret = new List<ApiDepartment>();
+        var departments = await DispatchAsync(new GetAllDepartments());
 
-        // Query
-        var departments = (await DispatchAsync(new GetAllDepartments())).ToArray();
-
-        if (departments.Length == 0)
-            return NotFound();
-        else
-        {
-            foreach (var d in departments) ret.Add(ApiDepartment.FromQueryDepartment(d));
-        }
+        var apiDepartments = departments.Select(ApiDepartment.FromQueryDepartment);
 
-        return Ok(ret);
+        return Ok(apiDepartments);
```

Comment:
If list is empty, should we not return no content or is that reserved when nothing (null) is "found"? The endpoint only returns 200 OK

Reply:
You're correct that a 404 status code typically indicates that a resource or endpoint is not found. Since the endpoint in question is valid and functioning as expected, returning a 404 would be misleading. 

When an API endpoint is accessed and successfully processes the request but finds no content to return, it's more appropriate to return a 204 No Content status code if you want to explicitly indicate that the operation was successful but there's no content to send back. However, returning a 200 OK with an empty array is also acceptable and can be seen as an indication that the request was successfully processed and the expected response is an empty list. This can be particularly useful when the client expects a list and can handle an empty list as a normal response.